AI-and-Analytics/Getting-Started-Samples/IntelPyTorch_GettingStarted/README.md

```diff
@@ -24,19 +24,40 @@ This Hello World sample code is implemented for CPU using the Python language.
  - The model is inherited from `torch.nn.Module`.
  - For the inference portion, `to_mkldnn()` function in `torch.utils.mkldnn` can accelerate performance by eliminating data reorders between operations, which are supported by Intel(R) DNNL.
 
-## License  
+## License
 Code samples are licensed under the MIT license. See
 [License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
 
 Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
 
-## How to Build and Run 
+## How to Build and Run
 ### Running Samples In DevCloud (Optional)
-If running a sample in the Intel DevCloud, please follow the below steps to build the python environment. Also, remember that you must specify the compute node (CPU, GPU, FPGA) and whether to run in batch or interactive mode. For more information, see the [Intel® oneAPI Base Toolkit Get Started Guide](https://devcloud.intel.com/oneapi/get-started/base-toolkit/) 
+
+<!---Include the next paragraph ONLY if the sample runs in batch mode-->
+### Run in Batch Mode
```

Comment:
Those are general contents for all samples.
Could you move out the contents from specific sample's README and move it to a general place such as README for all AI samples?

ex: https://github.com/oneapi-src/oneAPI-samples/blob/master/AI-and-Analytics/README.md